Fix HeMIS related bugs (#541)
ivadomed/ivadomed: docs/source/configuration_file.rst

-  ``missing_probability_growth``: Float. Controls missing probability
   growth at each epoch: at each epoch, the ``missing_probability`` is
   modified with the exponent ``missing_probability_growth``.
- ``contrasts``: List. List of contrasts used as input.
- ``ram``. Bool. Indicates whether to save the data in ram or not.

> - ``hdf5_path``: Str. Path to hdf5 files containing the dataset.

Reviewer: ```suggestion
- ``hdf5_path``: Str. Path to hdf5 file containing the dataset.
```

> - ``csv_path``: Str. Path to dataframe.

Reviewer: ```suggestion
- ``csv_path``: Str. Path to dataframe (csv file).
```

> - ``target_contrast``: Str. Contrast label used for ground truth.

Reviewer: ```suggestion
- ``target_contrast``: Str. Contrast whom the label file is used as ground truth.
```

> - `` Contrast label used for ROI cropping.``: Str. Contrast label used for ROI cropping.

Reviewer: ```suggestion
- `` Contrast label used for ROI cropping.``: Str. Contrast whom the label file is used for ROI cropping.
```

> - `` Contrast label used for ROI cropping.``: Str. Contrast whom the label file is used for ROI cropping.

Author: ```suggestion
- ``roi_contrast``: Str. Contrast whom the label file is used for ROI cropping.
```

-  ``missing_probability_growth``: Float. Controls missing probability
   growth at each epoch: at each epoch, the ``missing_probability`` is
   modified with the exponent ``missing_probability_growth``.

> - ``contrasts``: List. List of contrasts used as input.

Author: Add missing doc
